.vsts-ci.yml

```diff
@@ -104,7 +104,7 @@ stages:
               InputType: Basic
               Function: analyze
               TargetPattern: binskimPattern
-              AnalyzeTargetBinskim: $(Build.SourcesDirectory)\artifacts\bin\Sign.Cli\$(_BuildConfig)\net6.0\win10-x64\publish\*.dll
+              AnalyzeTargetBinskim: $(Build.SourcesDirectory)\artifacts\bin\Sign.Cli\$(_BuildConfig)\net8.0\win-x64\publish\*.dll
```

Comment:
[.NET 8 simplified RID's](https://learn.microsoft.com/dotnet/core/compatibility/sdk/8.0/rid-graph) such that `win-x64` replaced `win10-x64`.